lib/db/transactions.go

```diff
@@ -65,25 +64,25 @@ func (t readOnlyTransaction) getFileTrunc(key []byte, trunc bool) (FileIntf, boo
 	return f, true
 }
 
-func (t readOnlyTransaction) getGlobalInto(gk, dk, folder, file []byte, truncate bool) ([]byte, []byte, FileIntf, bool) {
-	gk = t.db.keyer.GenerateGlobalVersionKey(gk, folder, file)
+func (t readOnlyTransaction) getGlobal(buf, folder, file []byte, truncate bool) ([]byte, FileIntf, bool) {
```

Comment:
So `buf` in the calling function is fine, but calling it `buf` in here makes no sense. Maybe as the parameter name it's ok, because it's a random buffer of sorts and not actually the key "on the way in" - but then the assign on the next line should be `key :=` and that's what we use in the rest of the function and return. Other things like `t.Get(buf, nil)` looks like we're getting stuff into the buf or whatever.

Reply:
Actually, what about calling the parameter `key` instead of `buf`? That is unspecific (not gk, dk, nk, ...) and doesn't have the potential for the "fill-this-buffer" confusion.

Reply:
I think `keyBuf` is the better one then. It indicates a buffer that will hold a key. `key` looks like the caller should pass some kind of actual key there.